app/main.py

```diff
@@ -1,6 +1,29 @@
+import os
+from datetime import datetime as dt
+
+from dotenv import load_dotenv
+import requests
+
+load_dotenv()
+BASE_URL = "https://api.weatherapi.com/v1/current.json?"
```

Comment:
According to the checklist, you should use constants for all of your fixed values. The base URL of your API is a constant, so it's better to declare it as a constant at the top of your file. You did it correctly.